docs/config/shared-options.md

```diff
@@ -235,7 +234,7 @@ export default defineConfig({
       scss: {
         additionalData: `$injectedColor: orange;`,
       },
-      styl: {
+      style: {
```

Comment:
This isn't a typo. It's config for the `.styl` file - Stylus.

Reply:
Ouch, sorry about that, never worked with .style files.